packages/account/src/Sections/Profile/PersonalDetails/personal-details.jsx

```diff
@@ -101,10 +90,16 @@ class PersonalDetailsForm extends React.Component {
                 ? request.tax_identification_number.trim()
                 : '';
         }
-        request.citizen = request.citizen ? getLocation(this.props.residence_list, request.citizen, 'value') : '';
-        request.place_of_birth = request.place_of_birth
-            ? getLocation(this.props.residence_list, request.place_of_birth, 'value')
-            : '';
+
+        if (request.citizen) {
+            request.citizen = request.citizen ? getLocation(this.props.residence_list, request.citizen, 'value') : '';
```

Comment:
the ternary is redundant here as the above if condition will catch the `request.citizen`